src/test/scala/akka/cluster/swissborg/EitherValues.scala

```diff
@@ -0,0 +1,26 @@
+package akka.cluster.swissborg
+
+import org.scalactic.source
+import org.scalatest.exceptions.{StackDepthException, TestFailedException}
+
+trait EitherValues {
```

Comment:
scalatest `EitherValues` is not used because `either.right` is [deprecated in 2.13](https://www.scala-lang.org/api/2.13.1/scala/util/Either.html#right:scala.util.Either.RightProjection[A,B]). 

This is a [known issue](https://github.com/scalatest/scalatest/issues/972#issuecomment-555961675) in ScalaTest, which they will hopefully fix soon